common/config/MeleeCodes.c

```diff
@@ -274,6 +276,50 @@ const MeleeCodeLineItem rulesetLineItem = {
 	rulesetOptions,
 };
 
+
+/***************************************
+ * Widecreen Options
+ ***************************************/
+const MeleeCodeOption widescreenOptionOff = {
+	1,
+	"Off",
+	0,
+	NULL,
+};
+
+const MeleeCodeOption widescreenOptionWide = {
+	2,
+	"Wide",
+	g_widescreen_size,
+	g_widescreen,
+};
+
+const MeleeCodeOption *widescreenOptions[MELEE_CODES_WIDESCREEN_OPTION_COUNT] = {
+	&widescreenOptionOff,
+	&widescreenOptionWide,
+};
+
+static const char *widescreenDescription[] = {
```

Comment:
Description could be simpler since there is only one code. Also I think this would be better as a simple on off and the description should only say what turning it on does.